lib/core-common/src/types.ts

```diff
@@ -32,6 +32,7 @@ export interface Webpack5BuilderConfig extends BuilderConfigObject {
   name: 'webpack5';
   options?: {
     fsCache?: boolean;
+    lazyCompilation?: boolean;
```

Comment:
Do we need to document this somewhere?

Reply:
I have a separate ticket to do this, do you want to hold the PR for it?